ipamd/datastore/data_store_test.go

```diff
@@ -288,14 +288,15 @@ func TestPodIPv4Address(t *testing.T) {
 	assert.Equal(t, ds.eniIPPools["eni-2"].AssignedIPv4Addresses, 0)
 
 	noWarmIPTarget := 0
+	noMinimumIPTarget := 0
 
 	// Should not be able to free this ENI
-	eni := ds.RemoveUnusedENIFromStore(noWarmIPTarget)
+	eni := ds.RemoveUnusedENIFromStore(noWarmIPTarget, noMinimumIPTarget)
 	assert.True(t, eni == "")
 
 	ds.eniIPPools["eni-2"].createTime = time.Time{}
 	ds.eniIPPools["eni-2"].lastUnassignedTime = time.Time{}
-	eni = ds.RemoveUnusedENIFromStore(noWarmIPTarget)
+	eni = ds.RemoveUnusedENIFromStore(noWarmIPTarget, noMinimumIPTarget)
 	assert.Equal(t, eni, "eni-2")
 
```

Comment:
Can you please add a couple new tests that validates the new behaviour when MINIMUM_IP_TARGET > 0 and when WARM_IP_TARGET is set and not set?

Reply:
Sure thing.